API.md

```diff
@@ -0,0 +1,59 @@
+# API
+
+- `GET` `/api/v2/buildings/`
+
+```
```

Comment:
Add the "file type" after the first series of ` ``` `
i.e. ` ```json `
It should give IDE formatting superpowers and add colours to everything